packages/in_app_purchase/in_app_purchase_storekit/example/shared/RunnerTests/Mocks.h

```diff
@@ -0,0 +1,94 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <StoreKit/StoreKit.h>
+#import "FIATransactionCache.h"
+#import "FLTMethodChannelProtocol.h"
+#import "FLTPaymentQueueHandlerProtocol.h"
+#import "FLTPaymentQueueProtocol.h"
+#import "FLTRequestHandlerProtocol.h"
+#import "FLTTransactionCacheProtocol.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+// An interface representing a stubbed DefaultPaymentQueue
+@interface TestPaymentQueue : NSObject <FLTPaymentQueueProtocol>
+// FLTPaymentQueueProtocol properties
+@property(assign, nonatomic) SKPaymentTransactionState paymentState;
+@property(strong, nonatomic, nullable) id<SKPaymentTransactionObserver> observer;
+@property(strong, nonatomic, readwrite) SKStorefront *storefront API_AVAILABLE(ios(13.0));
+@property(strong, nonatomic, readwrite) NSArray<SKPaymentTransaction *> *transactions API_AVAILABLE(
+    ios(3.0), macos(10.7), watchos(6.2), visionos(1.0));
+
+// Test Properties
+@property(assign, nonatomic)
+    SKPaymentTransactionState testState;  // Set this property to set a test Transaction state, then
+                                          // call addPayment to add it to the queue.
+@property(strong, nonatomic, nonnull)
+    SKPaymentQueue *realQueue;  // This is a reference to the real SKPaymentQueue
+
+// Stubs
+@property(nonatomic, copy, nullable) void (^showPriceConsentIfNeededStub)(void);
+@property(nonatomic, copy, nullable) void (^restoreTransactionsStub)(NSString *);
+@property(nonatomic, copy, nullable) void (^startObservingPaymentQueueStub)(void);
+@property(nonatomic, copy, nullable) void (^stopObservingPaymentQueueStub)(void);
+@property(nonatomic, copy, nullable) void (^presentCodeRedemptionSheetStub)(void);
+@property(nonatomic, copy, nullable)
+    NSArray<SKPaymentTransaction *> * (^getUnfinishedTransactionsStub)(void);
+
+@end
+
+#pragma mark TransactionCache
```

Comment:
nit: either use `#pragma mark -` to create a divider, or just remove this mark since the class should already be shown in the dropdown list.

Reply:
okay, removed